src/plugins/lan/auth.c

```diff
@@ -126,7 +126,16 @@ uint8_t * ipmi_auth_md5(struct ipmi_session * s, uint8_t * data, int data_len)
  * Use OpenSSL implementation of MD2 algorithm if found.
  * This function is analogous to ipmi_auth_md5
  */
-uint8_t * ipmi_auth_md2(struct ipmi_session * s, uint8_t * data, int data_len)
+uint8_t * ipmi_auth_md2(
+#ifdef HAVE_CRYPTO_MD2
```

Comment:
To reduce duplication, please define a macro like:
```
#ifdef HAVE_CRYPTO_MD2
  #define __MD2_ONLY__(x) x
#else 
  #deine __MD2_ONLY__(x) __UNUSED__(x)
#endif 
```
and use it with the arguments of the function. That way you will have each argument declared only once.